File: core/test/de/fatox/meta/test/MetaTest.java
package de.fatox.meta.test;

import com.badlogic.gdx.backends.headless.HeadlessApplication;
import com.badlogic.gdx.backends.headless.HeadlessApplicationConfiguration;
import de.fatox.meta.Meta;
import org.junit.Before;

public abstract class MetaTest {

    @Before
    public void prepare() {
        final HeadlessApplicationConfiguration config = new HeadlessApplicationConfiguration();
        new HeadlessApplication(new TestApp(), config);
        Meta.inject(this);
    }
}

File: runtime/src/test/kotlin/de/fatox/meta/test/MetaTest.kt
package de.fatox.meta.test

import com.badlogic.gdx.backends.headless.HeadlessApplication

abstract class MetaTest {
    @Before
    fun prepare() {
        val config = HeadlessApplicationConfiguration()
        HeadlessApplication(TestApp(), config)
        Meta.inject(this)
    }
}